lib/ansible/modules/monitoring/zabbix/zabbix_screen.py

```diff
@@ -350,16 +351,24 @@ def main():
             http_login_password=dict(type='str', required=False, default=None, no_log=True),
             validate_certs=dict(type='bool', required=False, default=True),
             timeout=dict(type='int', default=10),
-            screens=dict(type='list', elements='dict', required=True, options=dict(
-                screen_name=dict(type='str', required=True),
-                host_group=dict(type='str'),
-                state=dict(type='str', default='present', choices=['absent', 'present']),
-                graph_names=dict(type='list', elements='str'),
-                graph_width=dict(type='int', default=None),
-                graph_height=dict(type='int', default=None),
-                graphs_in_row=dict(type='int', default=3),
-                sort=dict(default=False, type='bool'),
-            ))
+            screens=dict(
+                type='list',
+                elements='dict',
+                required=True,
+                options=dict(
```

Comment:
```suggestion
                options=dict(
```
could you please leave this as it was too for this PR. Let's restrict it with only formatting changes

Reply:
ok, i see, it was there in the doc. I didn't use this syntax before.
Thank you